spec/griddler/sendgrid/adapter_spec.rb

```diff
@@ -14,6 +14,14 @@
       to: 'Hello World <hi@example.com>',
       cc: 'emily@example.com',
       from: 'There <there@example.com>',
+      charsets: {
+        to: 'UTF-8',
+        cc: 'UTF-8',
+        subject: 'UTF-8',
+        from: 'UTF-8',
+        html: 'UTF-8',
+        text: 'iso-8859-1'
```

Comment:
You don't gain much in this test by having an exhaustive list of `charsets`. I suggest only having 2, with different values. Then you can do away with the loop in the spec.